securesystemslib/formats.py

```diff
@@ -237,10 +237,11 @@
   keyval = KEYVAL_SCHEMA,
   expires = SCHEMA.Optional(ISO8601_DATETIME_SCHEMA))
 
-# Like KEY_SCHEMA, but requires keyval's private portion to be not set or empty
+# Like ANYKEY_SCHEMA, but requires keyval's private portion to be not set or empty
```

Comment:
NOTE: I don't see any major issues with this pull request, but I will make minor edits to your pull request for any minor issues.  I will share my in-line comments on this pull request for documentation purposes.

This SCHEMA is more like KEY_SCHEMA, since ANYKEY_SCHEMA contains an attribute (keyid) that ANYKEY_SCHEMA doesn't.  PUBLIC_KEY_SCHEMA matches KEY_SCHEMA, and will now differ with the `keyid_hash_algorithms` and `keyval` attributes.